src/main/scala/org/broadinstitute/dsde/firecloud/service/TrialService.scala

```diff
@@ -0,0 +1,90 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import akka.actor.{Actor, Props}
+import akka.pattern._
+import com.typesafe.scalalogging.slf4j.LazyLogging
+import org.broadinstitute.dsde.firecloud.Application
+import org.broadinstitute.dsde.firecloud.dataaccess.{SamDAO, ThurloeDAO}
+import org.broadinstitute.dsde.firecloud.model.Trial.{TrialStates, UserTrialStatus}
+import org.broadinstitute.dsde.firecloud.model.{RequestCompleteWithErrorReport, UserInfo}
+import org.broadinstitute.dsde.firecloud.service.PerRequest.{PerRequestMessage, RequestComplete}
+import org.broadinstitute.dsde.firecloud.service.TrialService.{EnableUser, EnrollUser, TerminateUser}
+import spray.http.StatusCodes._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object TrialService {
+  sealed trait TrialServiceMessage
+  case class EnableUser(userInfo:UserInfo) extends TrialServiceMessage
+  case class EnrollUser(userInfo:UserInfo) extends TrialServiceMessage
+  case class TerminateUser(userInfo:UserInfo) extends TrialServiceMessage
+
+  def props(service: () => TrialService): Props = {
+    Props(service())
+  }
+
+  def constructor(app: Application)()(implicit executionContext: ExecutionContext) =
+    new TrialService(app.samDAO, app.thurloeDAO)
+}
+
+class TrialService
+  (val samDao: SamDAO, val thurloeDao: ThurloeDAO)
+  (implicit protected val executionContext: ExecutionContext)
+  extends Actor with LazyLogging {
+
+  override def receive = {
+    case EnableUser(userInfo) => enableUser(userInfo) pipeTo sender
+    case EnrollUser(userInfo) => enrollUser(userInfo) pipeTo sender
+    case TerminateUser(userInfo) => terminateUser(userInfo) pipeTo sender
+  }
+
+  // TODO: implement fully! Check that the user does not already have a state, before overwriting.
+  // this method exists solely for developer-testing purposes right now.
+  private def enableUser(userInfo: UserInfo): Future[PerRequestMessage] = {
+    // build the state that we want to persist to indicate the user is enabled
+    val now = System.currentTimeMillis
+    val enabledStatus = UserTrialStatus(userInfo.id, Some(TrialStates.Enabled), now, 0, 0, 0)
+    thurloeDao.saveTrialStatus(userInfo, enabledStatus) map { _ =>
+      RequestComplete(spray.http.StatusCodes.OK)
+    }
+  }
```

Comment:
As a developer running your own local instance of orchestration, you can call `enableUser` on yourself by hacking the code ... which puts you in a position for `enrollUser` to succeed.